composer.json

```diff
@@ -21,27 +21,32 @@
         "CQRS",
         "DDD"
     ],
+    "minimum-stability": "dev",
+    "prefer-stable": true,
     "require": {
-        "php": "~5.5|~7.0",
-        "beberlei/assert": "^2.0",
-        "prooph/common" : "^3.3",
-        "react/promise" : "^2.2"
+        "php": "~5.5 || ~7.0",
+        "beberlei/assert": "^2.4",
+        "prooph/common" : "^3.7",
+        "react/promise" : "^2.2.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.8",
-        "fabpot/php-cs-fixer": "1.7.*",
+        "phpunit/phpunit": "^4.8.23",
+        "fabpot/php-cs-fixer": "^1.7",
         "satooshi/php-coveralls": "^1.0",
         "container-interop/container-interop" : "^1.1",
-        "sandrokeil/interop-config": "^0.3",
-        "tobiju/bookdown-bootswatch-templates": "^0.2.0"
+        "sandrokeil/interop-config": "^1.0",
+        "tobiju/bookdown-bootswatch-templates": "^1.0"
     },
     "suggest": {
         "prooph/event-store": "Let the EventBus dispatch persisted DomainEvents",
-        "zendframework/zend-servicemanager": "Use Zf2 ServiceManager to lazy load your handlers and listeners",
+        "zendframework/zend-servicemanager": "Use Zend ServiceManager to lazy load your handlers and listeners",
         "prooph/service-bus-zfc-rbac-bridge": "Use ZfcRbac as authorization service for route and finalize guard",
         "container-interop/container-interop": "For usage of provided factories",
         "sandrokeil/interop-config": "For usage of provided factories"
     },
+    "conflict": {
+        "sandrokeil/interop-config": "<1.0"
```

Comment:
Ah ok, it's only require dev. Ok then.
Am 06.03.2016 21:59 schrieb "Sandro Keil" <notifications@github.com>:

> In composer.json
> <https://github.com/prooph/service-bus/pull/116#discussion_r55149316>:
>
> >          "prooph/service-bus-zfc-rbac-bridge": "Use ZfcRbac as authorization service for route and finalize guard",
> >          "container-interop/container-interop": "For usage of provided factories",
> >          "sandrokeil/interop-config": "For usage of provided factories"
> >      },
> > +    "conflict": {
> > +        "sandrokeil/interop-config": "<1.0"
>
> We have no version constraint for interop-config. If we want to release a
> minor version and people want to upgrade, they must also upgrade their
> dependency.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prooph/service-bus/pull/116/files#r55149316>.
>